spec/requests/api/v1/documents_spec.rb

```diff
@@ -43,14 +43,17 @@
     end
 
     let(:valid_params) do
-      { document_id: id,
-        title:       'my title',
-        path:        'http://www.gov.gov/goo.html',
+      {
+        document_id: id,
```

Comment:
I can see that this is the spec that ran afoul of the `Layout/HashAlignment` cop. While I agree that the previous version is easier to read, I'm inclined to stick with the current configuration, which is the Rubocop default. There are a couple of issues with the "align both keys and values" style:
- it means that any change to the hash has the potential to require a re-alignment of all the other keys. Rubocop could do that for us automagically, but...
- the other downside is that it muddies the git history. It's harder to track the source and rational for changes when the hash has changed many times.
- most of the hashes in our various repos use the default format, so changing it would result in even more Rubocop offenses, across repos.

TL;DR: thanks for making these changes, even if you had to hold your nose.